lib/widget/button.dart

```diff
@@ -125,14 +125,18 @@ class ButtonState extends WidgetState<Button> {
     if (isOutlineButton) {
       rtn = BoxWrapper(
         boxController: widget.controller,
-        widget: OutlinedButton(
+        ignoresPadding: true,
+        ignoresMargin: true,
+        widget: TextButton(
```

Comment:
Sneh changed the button from TextButton to OutlinedButton previously. @snehmehta this is ok to be back to TextButton?

Reply:
Yup good to go